Userland/Utilities/strace.cpp

```diff
@@ -896,9 +896,9 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         syscall_arg_t arg3 = regs.rbx;
 #elif ARCH(AARCH64)
         syscall_arg_t syscall_index = 0; // FIXME
-        syscall_arg_t arg1 = 0; // FIXME
-        syscall_arg_t arg2 = 0; // FIXME
-        syscall_arg_t arg3 = 0; // FIXME
+        syscall_arg_t arg1 = 0;          // FIXME
+        syscall_arg_t arg2 = 0;          // FIXME
+        syscall_arg_t arg3 = 0;          // FIXME
```

Comment:
Please squash this into the commit where it was added

Reply:
Oops, fixed. :)